competencies/roles/DeveloperLevelMapping.md

```diff
@@ -4,8 +4,10 @@ The three categories of competencies can be used to map to typical job titles sp
 
 | Title/Job           | Key Competencies           | Other Competencies           |     Notes  |
 | ------------------- |:-------------------------:| ----------------------------:| ----------:|
-| SW Dev I (junior)   |                           | 90% emerging; 10% proficient |  New Grad, or first one or two years |
-| SW Dev II           | Proficient: "Tech Design, Implementation and Code Ownership"      |  50% emerging, 50% proficient  |   |
-| SW Dev III (senior) | Proficient: "Communication" and "System Design" | 80% proficient; some authority  | May be expert or generalist | 
-| Principle SW Dev    | Proficient: "Technical Change" | 50% Proficient; 50% Authority | Strong self awareness, Strong on general areas such as business acumen | 
-| Distinguished       | Authority: "Technical Change" | | Broad authority or extreme depth in a technical branch | 
+| SW Dev I (junior)   |   |   |  New Grad, or first one or two years |
+| SW Dev II           |   |  75% emerging, 25% proficient  |   |
+| SW Dev III (intermediate) | Proficient: "Tech Design, Implementation and Code Ownership" | 50% emerging, 50% proficient |   |
+| SW Dev IV (senior I) | Proficient: "Communication" or "System Design" | 50% proficient; some authority  |   |
```

Comment:
I didn't realize this with the original chart but this seems like the biggest space between levels.